api_tests/nodes/views/test_node_alternative_citations.py

```diff
@@ -0,0 +1,1098 @@
+        res = self.app.patch_json_api(self.citation.private_url, self.payload(name='new name', id=self.citation._id), auth=self.user_three.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.citation.reload()
+        assert_equal(self.citation.text, "text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_text_only_admin_public(self):
+        res = self.app.patch_json_api(self.citation.public_url, self.payload(text='new text', id=self.citation._id), auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        self.citation.reload()
+        assert_equal(self.citation.text, "new text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_text_only_admin_private(self):
+        res = self.app.patch_json_api(self.citation.private_url, self.payload(text='new text', id=self.citation._id), auth=self.user.auth)
+        assert_equal(res.status_code, 200)
+        self.citation.reload()
+        assert_equal(self.citation.text, "new text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_text_only_non_admin_public(self):
+        res = self.app.patch_json_api(self.citation.public_url, self.payload(text='new text', id=self.citation._id), auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.citation.reload()
+        assert_equal(self.citation.text, "text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_text_only_non_admin_private(self):
+        res = self.app.patch_json_api(self.citation.private_url, self.payload(text='new text', id=self.citation._id), auth=self.user_two.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.citation.reload()
+        assert_equal(self.citation.text, "text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_text_only_non_contrib_public(self):
+        res = self.app.patch_json_api(self.citation.public_url, self.payload(text='new text', id=self.citation._id), auth=self.user_three.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.citation.reload()
+        assert_equal(self.citation.text, "text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_text_only_non_contrib_private(self):
+        res = self.app.patch_json_api(self.citation.private_url, self.payload(text='new text', id=self.citation._id), auth=self.user_three.auth, expect_errors=True)
+        assert_equal(res.status_code, 403)
+        self.citation.reload()
+        assert_equal(self.citation.text, "text")
+        assert_equal(self.citation.name, "name")
+
+    def test_update_citation_name_only_repeat_admin_public(self):
+        res = self.app.put_json_api(self.citation.public_url, self.payload(name="name2", id=self.citation._id), auth=self.user.auth, expect_errors=True)
```

Comment:
Should this be a `patch` request? Please double check all of tests cases following this one. I think they're supposed to be `patch` requests because the payload does not include all of the fields.